### PR TITLE
Add className support to page container

### PR DESCRIPTION
**Pull request by DavidWells**
I need access to the page container to apply a class.

My use case is to apply some fancy min height css to the page container to make it fit in viewport always

```
.pageContainer {
 min-height:  min-height: calc(100vh - $navHeight - $footerHeight);
}
```

image

### CODE REVIEW

**Comment by DavidWells** [comment]
Hmmm I guess I can wrap my code: https://github.com/serverless/site/blob/master/src/app.js#L48 but I'd like to apply classes to phenomic div too https://github.com/serverless/site/blob/master/src/layouts.js#L15 =)

**Comment by thangngoc89** [comment]
@DavidWells try flexbox. it would be a lot easier 

https://philipwalton.github.io/solved-by-flexbox/demos/sticky-footer/

File: src/PageContainer/component.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ type Props = {
     splat: string,
   },
   layouts: Object,
+  className: string,
```
Comment (thangngoc89) [inline src/PageContainer/component.js]: We don't want this a requirement. Try `className?: string`

**Comment by MoOx** [comment]
@DavidWells I am not against this change, but indeed, flexbox is the best solution here. You can see it in action here https://phenomic.io/docs/
I am also going to update the default boilerplate (wip) to have this by default (body take full height - header - footer).

**Comment by thangngoc89** [comment]
IMHO, we should allow user to pass their props to PageContainer. How about `{...remainingProps}`. This way, we don't have to merge PR for each specific case like this

**Comment by MoOx** [comment]
Yeah good idea!

**Comment by DavidWells** [comment]
@MoOx not sure if people will want that default flex or not. Aka not sure of side effects it might cause for people who want ugly sites =PP

I'd just spread the props into <PageContainer/>

**Pull request closed without merging**